pom.xml

```diff
@@ -200,225 +249,305 @@
     </build>
     <dependencies>
         <dependency>
+            <!-- Plexus Common Utilities -->
+            <groupId>org.codehaus.plexus</groupId>
+            <artifactId>plexus-utils</artifactId>
+            <version>3.3.0</version>
+            <type>jar</type>
+            <scope>compile</scope>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <!-- Swing Layout Extensions -->
+            <groupId>org.swinglabs</groupId>
+            <artifactId>swing-layout</artifactId>
+            <version>1.0.3</version>
+            <type>jar</type>
+            <scope>compile</scope>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <!-- Gson (because Spigot is behind in their version) -->
```

Comment:
Doesn't matter if Spigot is behind, unless we're going to shade this version in (which wouldn't make sense) we should just use the one that spigot supplies.